Chargebee/Classes/ReceiptValidation/CBValidateReceiptResource.swift

```diff
@@ -74,6 +74,9 @@ struct PayloadBody: URLEncodedRequestBody {
     let customerId: String
     let period: String
     let periodUnit: String
+    let first_name: String
+    let last_name: String
+    let email: String
```

Comment:
Please rename this to use camelCase

Reply:
we are sending params to backend by endcoding